yarn-project/end-to-end/src/e2e_blacklist_token_contract/minting.test.ts

```diff
@@ -87,7 +87,7 @@ describe('e2e_blacklist_token_contract mint', () => {
         const receipt = await asset.methods.mint_private(amount, secretHash).send().wait();
         txHash = receipt.txHash;
 
-        await t.addPendingShieldNoteToPXE(0, amount, secretHash, txHash);
+        await t.addPendingShieldNoteToPXE(asset, wallets[0], amount, secretHash, txHash);
```

Comment:
What is a PendingShieldNote?
The changes you're making are consistent with the code that was here already, so to review these e2e tests I don't actually need to understand what the tests are trying to do, but I am wondering.